### ISSUE TITLE
Missing alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [itsshala/dasmoto#3]
It looks like you're missing the alt attribute for this image. 

It's best practice to include the alt attribute for every image on your page considering it is used by screen readers for [accessibility](https://www.codecademy.com/articles/accessibility) and as a back up in case your image cannot be rendered. 

For example: `        <img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-2/hacksaw.jpeg" alt="Hacksaw"/>
`

https://github.com/itsshala/dasmoto/blob/aec31ec12df027764846631b735891f48f24ec56/index.html#L15
